CondFormats/GeometryObjects/interface/PDetGeomDesc.h

```diff
@@ -2,25 +2,41 @@
 #define CondFormats_PDetGeomDesc_h
 
 #include "CondFormats/Serialization/interface/Serializable.h"
+#include "Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h"
+#include "DetectorDescription/DDCMS/interface/DDTranslation.h"
+#include "DetectorDescription/DDCMS/interface/DDRotationMatrix.h"
 
 #include <vector>
 #include <string>
 
 class PDetGeomDesc {
 public:
-  PDetGeomDesc(){};
-  ~PDetGeomDesc(){};
-
   struct Item {
+    Item() = default;
```

Comment:
Prior to signing off for AlCa, I would like @ggovi to sign off for DB on changes to classes/structs that use the `COND_SERIALIZABLE` macro.

Reply:
@christopheralanwest 
No change on the persistent data members, so no objection on this respect...